src/clients/iscsi-client_finish.rb

```diff
@@ -76,7 +76,7 @@ def main
         # write open-iscsi database of automatic connected targets
         WFM.Execute(
           path(".local.bash"),
-          "/usr/bin/test -d /etc/iscsi/ && " \
+          "test -d /etc/iscsi/ && " \
```

Comment:
Here I'd replace it with native Ruby code and avoid any shell calls:

```ruby
if File.directory?("/etc/iscsi/")
  target = File.join(Installation.destdir, "etc/iscsi")
  ::FileUtils.mkdir_p(target)
  # the preserve flag is equivalent to `cp -a`, `/.` copies content of directory instead of the directory itself
  # see https://ruby-doc.org/stdlib-2.5.0/libdoc/fileutils/rdoc/FileUtils.html#method-c-cp_r
  ::FileUtils.cp_r("/etc/iscsi/.", target, preserve: true)
end
```

Reply:
That's fine with me. Thanks!